linux/sockstat.go

```diff
@@ -0,0 +1,94 @@
+package linux
+
+import (
+	"io/ioutil"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+type Sockstat struct {
+	// sockets:
+	SocketsUsed uint64 `json:"sockets_used" altname:"sockets:used"`
```

Comment:
Please change the tag format to `field:"sockets.used"`, `field:"TCP.inuse"` like that.